collection-service/database.js

```diff
@@ -0,0 +1,95 @@
+import mysql from 'mysql2';
```

Comment:
do u think it's better to rename this file to collection-controller which is common for node js backend?
also, this file contains functions to manipulate the database and is not the database itself?

Reply:
yep. actually i want to refactor (yikes) both book-service and collection-service tgt since their structure are quite similar so how about creating another PR for that?

Reply:
Sure can